xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_version.xsd

```diff
@@ -655,6 +655,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Visibible Indication of parking area.  +v1.2.2</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="MonitoredAvailability" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether real-time global availability data is normally available for the whole PARKING or PARKING AREA.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="MonitoredBays" type="xsd:boolean" minOccurs="0">
```

Comment:
Why is this important to be added?

Reply:
There are multiple use cases: some people prefer a Parking where free spots can be easily identified, or for disabled reserved bays, to know if these ones are available or not, etc.

Reply:
Can we identify individual bays in NeTEx? If so, why not apply monitored on them instead of the aggregated here?

Reply:
Is this then still a boolean? Or something like true/false/partial.

Reply:
My interpretation would be: *MonitoredAvailabilty* tells whether a real-time service exists; if *true*, then *MonitoredBays* would tell whether the granularity of the real-time service is a the level of individual bays or less precise.

Reply:
What if only half of the individual bays would be monitored?